src/i18n/en.json

```diff
@@ -70,7 +70,9 @@
     "addInfo": "Additional info",
     "schoolName": "School name",
     "schoolClass": "School name",
+    "language": "ENGLISH",
```

Comment:
This doesn't seem like the right place to store this information. I am assuming that you are using this translation to set the user's language somewhere.